client/util/time.ts

```diff
@@ -2,27 +2,36 @@
 import lang from '../lang'
 
 export function secondsToTime(s: number): string {
-    const divide = [60, 24, 30, 12];
-    const unit = ['minute', 'hour', 'day', 'month'];
-    let time = s / 60;
+    const divide = [60, 24, 30, 12]
+    const unit = ['minute', 'hour', 'day', 'month']
```

Comment:
Why not jut add `"seconds"` here (`unit` and `divide`) and skip the initial division by 60 instead of special-casing seconds?

Reply:
Already tried that, didn't work without retarded changes.

Reply:
Actually, let me look into that once more.